src/interaction/useInteractionMode.ts

```
@@ -0,0 +1,106 @@
+  return window.matchMedia(query).matches;
+};
+
+const getCapabilityState = (): Pick<InteractionModeState, "canHover" | "hasFinePointer" | "hoverNone"> => ({
+  canHover: getMediaMatch("(any-hover: hover)"),
+  hasFinePointer: getMediaMatch("(any-pointer: fine)"),
+  hoverNone: getMediaMatch("(hover: none)"),
+});
+
+const emit = (nextState: InteractionModeState) => {
+  cachedState = nextState;
+  subscribers.forEach((subscriber) => subscriber(cachedState));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)